src/Library/demos/Picture/main.blp

```diff
@@ -0,0 +1,49 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Picture");
+  description: _("A widget that displays images at their natural sizes");
+
+  child: Box {
+    orientation: vertical;
+
+    Picture picture {
+      halign:center;
+      valign:center;
+      can-shrink: bind can_shrink_switch.active;
+    }
+
+    Adw.Clamp {
+      maximum-size: 500;
+
+      Adw.PreferencesGroup {
+        title: _("Picture Properties");
```

Comment:
I don't think the title is useful, there is no ambiguity to clarify

You can use a `ListBox` with `styles ["boxed-list"]` instead